RSFormView/Classes/Models/FormConfigurator.swift

```diff
@@ -21,6 +21,13 @@ public class FormConfigurator {
   public var textColor = UIColor.brightGray
   public var errorTextColor = UIColor.red
   
+  public var validBorderColor = UIColor.lightGray
+  public var invalidBorderColor = UIColor.red
+  public var editingBorderColor = UIColor.astralBlue.withAlphaComponent(0.8)
+  
+  public var borderCornerRadius = CGFloat(8)
```

Comment:
`explicit type is preferred: `public var borderCornerRadius: CGFloat = 8

Reply:
is this just for CGFloat though ? because we are inferring the type in most properties

Reply:
Here it says for single instances: